bin/update-contact-email

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+# Updates a contact's email address in forms so it will be reflected in solr and in future bulk emails
+# The change will be reflected in solr when the index-users-in-solr cron job runs (every minute)
+#
+# Usage: update-contact-email old_email new_email
+
+require_relative '../deployment'
+require 'json'
+require 'cdo/db'
+
+def update_form_data(raw, new_email)
+  json = JSON.parse raw
+  json[:email_s] = new_email
+  json.to_s
+end
+
+def update_email(old_email, new_email)
+  forms = PEGASUS_DB[:forms].where(email: old_email).all
+  puts "Email #{old_email} found on #{forms.length} forms:"
+  forms.each do |form|
+    puts "  #{form[:id]}: #{form[:kind]}"
+  end
+  puts "Update #{old_email} with #{new_email}? (y/N)"
+
+  unless STDIN.readline.strip.downcase == 'y'
+    puts 'Canceled.'
+    exit
+  end
+
+  PEGASUS_DB.transaction do
+    forms.each do |form|
+      new_data = update_form_data(form[:data], new_email)
+      PEGASUS_DB[:forms].where(id: form[:id]).update(email: new_email, data: new_data)
```

Comment:
Set indexed_at to null. @ashercodeorg, can you elaborate on your concern there? Why is it dangerous? How often is "too often".